### PR TITLE
Lua beautifier

### LINKED ISSUE #161: Support for Lua
Hello, any chance you will be adding lua as a supported language?

### PR DESCRIPTION
**Pull request by JoostvDoorn**
### What does this implement/fix? Explain your changes.

Adds a perl script beautifier for lua, by @pkulchenko

### Does this close any currently open issues?

Fixes #161 

### Any other comments?

Edit: So I am not sure how this plays around with settings in atom-beautify, so if the developers of this package could check this.

### Checklist

Check all those that are applicable and complete.

- [X] Merged with latest `master` branch
- [X] Added examples for testing to [examples/ directory](examples/)
- [ ] Travis CI passes (Mac support)
- [ ] AppVeyor passes (Windows support)

### CODE REVIEW

File: spec/beautify-languages-spec.coffee [hunk 1]
```diff
@@ -26,7 +26,7 @@ describe "BeautifyLanguages", ->
     "mustache", "objective-c", "perl", "php",
     "python", "ruby", "sass", "sql", "svg",
     "xml", "csharp", "gfm", "marko",
-    "go", "html-swig"
+    "go", "html-swig", "lua"
```
Comment (Glavin001) [inline spec/beautify-languages-spec.coffee]: You will need to install `language-lua` package as well. See https://github.com/Glavin001/atom-beautify/blob/master/.travis.yml#L25

**Comment by Glavin001** [comment]
Tests are currently failing. Once they pass, this can be merged. Thanks!

**Comment by JoostvDoorn** [comment]
I added language-lua to the travis file, and squashed the commits.

**Comment by Glavin001** [comment]
Sorry for the delay! Please merge with master so I can merge. Thanks!

**Comment by JoostvDoorn** [comment]
I just rebased and pushed, seems tests fail though. Will have to look at this later.

**Comment by Glavin001** [comment]
Thank you for contributing!

**Comment by Glavin001** [comment]
Published to v0.29.9

**Pull request merged**